src/library/rekordbox/rekordboxfeature.cpp

```diff
@@ -244,7 +244,7 @@ void insertTrack(
     QString year = QString::number(track->year());
     QString genre = genresMap[track->genre_id()];
     QString location = devicePath + getText(track->file_path());
-    float bpm = static_cast<float>(track->tempo() / 100.0);
+    float bpm = static_cast<float>(track->tempo()) / 100.0f;
```

Comment:
Before: Calculation in double, result cast to float
After: Calculation in float
Does this really make a difference? Then maybe leaving a comment for the future would be a good idea.

Reply:
My mistake, this actually wasn't a problem. Have reverted.